flow/alerting/classifier.go

```diff
@@ -692,6 +692,15 @@ func GetErrorClass(ctx context.Context, err error) (ErrorClass, ErrorInfo) {
 		}
 	}
 
+	// MongoDB can leak error without properly encapsulate it into a pre-defined error type
+	// so here we use string matching as a catch-all to avoid false alarms
+	if strings.Contains(err.Error(), "(ShutdownInProgress) The server is in quiesce mode and will shut down") {
```

Comment:
`"(ShutdownInProgress) The server is in quiesce mode and will shut down"`
can push to a common constant